kiosk/src/Models/Kiosk.ts

```diff
@@ -122,6 +122,18 @@ export class Kiosk {
         }
     }
 
+    removeGameFromList(): void {
+        const removedGame: GameData = this.games.splice(this.selectedGameIndex!, 1)[0];
+        const addedGameIds = this.getAllAddedGames();
+        console.log(addedGameIds);
+        delete addedGameIds[removedGame.id];
+
+        if (removedGame.id in addedGameIds) {
+            localStorage.removeItem(this.addedGamesLocalStorageKey);
+        }
```

Comment:
re: discussion offline, it looks like this method is not used & should be deleted~